src/main/java/org/dimdev/vanillafix/bugs/mixins/client/MixinEntityPlayerSP.java

```diff
@@ -14,8 +13,8 @@
      * vanilla code thinks no GUI is open by forcing Minecraft.currentScreen to
      * always return null. (see https://bugs.mojang.com/browse/MC-2071)
      */
-    @Redirect(method = "onLivingUpdate", at = @At(value = "FIELD", target = "Lnet/minecraft/client/Minecraft;currentScreen:Lnet/minecraft/client/gui/GuiScreen;", ordinal = 0))
-    private GuiScreen getCurrentScreen(Minecraft mc) {
-        return null;
+    @Redirect(method = "onLivingUpdate", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/gui/GuiScreen;doesGuiPauseGame()Z"))
+    private boolean onPauseCheck(GuiScreen guiScreen) {
+        return false;
```

Comment:
Did you test this (can you open chat while in a nether portal)? I'm not sure how this works, wouldn't this treat all GUIs as non-pausing GUIs, making the bug even worse by not even allowing you to open the pause menu?

Maybe you meant `return true`?